### PR TITLE
Np 985 try with consumer

### PR DESCRIPTION
**Pull request by axthosarouris**
Implement forEach for Try. 

Functionality:
   forEach returns a `Success<Void>` if the operation is successful, or a Failure<Void> if the operation (or a previous chained operation) has failed.
 
Necessary for nva-user-access-service

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -1,19 +1,19 @@
...
-  - rm -f  $HOME/.gradle/caches/modules-2/modules-2.lock
-  - rm -f $HOME/.gradle/caches/*/plugin-resolution/
-cache:
-  directories:
-    - $HOME/.gradle/caches/
-    - $HOME/.gradle/wrapper/
-before_install:
-  - sudo apt-get install -y jq
-  - LATEST_CODACY_VERSION="$(curl -Ls https://api.bintray.com/packages/codacy/Binaries/codacy-coverage-reporter/versions/_latest | jq -r .name)"
+#before_cache:
```
Comment (janbj) [inline .travis.yml:4]: Is this meant to me commented out?

File: build.gradle [hunk 2]
```diff
@@ -62,6 +62,9 @@ dependencies {
     implementation group: 'com.fasterxml.jackson.core', name: 'jackson-annotations', version: jacksonVersion
     implementation group: 'org.apache.httpcomponents', name: 'httpcore', version: '4.4.11'
     implementation group: 'org.apache.httpcomponents', name: 'httpclient', version: '4.5.9'
+//    implementation(group: 'org.slf4j', name: 'slf4j-api', version: '1.8.0-beta4') {
```
Comment (janbj) [inline build.gradle:65]: Meant to be commented out?

File: .travis.yml (hunk 1 above)
Comment (axthosarouris) [reply .travis.yml:4]: No. these were actually accidentally pushed. I was about to merge the branch, but travis was failing for a mysterious reason.

File: src/main/java/nva/commons/utils/JsonUtils.java [hunk 3]
```diff
@@ -11,9 +11,8 @@
 import com.fasterxml.jackson.databind.deser.std.StdDeserializer;
 import com.fasterxml.jackson.databind.deser.std.StringDeserializer;
 import com.fasterxml.jackson.databind.module.SimpleModule;
-import java.io.IOException;
-
 import com.fasterxml.jackson.datatype.jsr310.JavaTimeModule;
+import java.io.IOException;
```
[the remark this answers is not in the document]
Comment (axthosarouris) [reply src/main/java/nva/commons/utils/JsonUtils.java:15]: Travis does not like it

**Pull request merged**